### ISSUE TITLE
mobcount with "All" event label as parameter.

### ISSUE THREAD

**Issue by emistry** [HerculesWS/Hercules#784]
## Issue :
```
mobcount( "prontera","All" )
```

## Error :
![Image of Yaktocat](http://i.imgur.com/z18Nm1s.png)
Erm, it doesn't seem like support "All".  
**It only support "all".**

**Comment by Aeromesi** [comment]
If you're trying to do a mobcount why not just declare the mob id of the count, and make a label that gets executed everytime you kill the certain mob that adds the counts? In my Zombie massacre thats how I made the mob counts

**Comment by emistry** [comment]
I don't want to use extra mobcount to count for each monster with different label.
I dont' want to use extra variable to store these mob count, it's not necessary, and ugly, and hard to maintain.
Just imagine what you going to do if you spawn 100 monsters with different amount + event labels.
well, that's not the main case here ayway.

mobcount with "all" parameter does solve all of this. The only problem here is that, the mobcount doesn't support the "all" parameter if the "a" is capital letter.

**Comment by Aeromesi** [comment]
Oh so what that code does is do a count using a certain function called "All" but it doesn't work?

**Comment by dastgir** [comment]
Case Sensitivity
mobcount( "prontera","All" )
Should be
mobcount( "prontera","all" )

**Comment by emistry** [comment]
Why I wanna express is that, why is "All" not acceptable ??

all other event label we start it using "OnEventLabel" ... **the first letter is Uppercase**
but why "all" is used and not "All" ???

**Comment by MishimaHaruna** [comment]
Well, the documentation in script_commands.txt clearly mentions `"all"`, not `"All"`, and it's been like that since January 2012 when the `"all"` special value was added. The reason why it's lowercase rather than uppercase might be to differentiate it from event labels (it's a special value, not an actual event!), seen how the other special value is `"this"` (again lowecase). Or there might be no reason at all, pure preference of the writer.
I'm fine with it being lowercase for the reason I suggested.
